### ISSUE TITLE
aria-progressbar-name - Ensures every ARIA progressbar node has an accessible name

### ISSUE THREAD

**Issue by JaneSjs** [surveyjs/survey-library#7067]
User Issue: T14878 - Progress Bar Node is missing an Accessible Name
https://surveyjs.answerdesk.io/internal/ticket/details/T14878
***
To reproduce the issue, navigate to the [Progress Indicators](https://surveyjs.io/form-library/examples/configure-form-navigation-with-progress-indicators/reactjs) demo, and run the Fast Pass check using the Accessibility Insights for Web tool. 

The following error appears: `aria-progressbar-name - Ensures every ARIA progressbar node has an accessible name`.

![image](https://github.com/surveyjs/survey-library/assets/22372972/a057d6a0-7744-4d8a-8f99-6a2f1a8fbba7)
